org.clementine_player.Clementine.appdata.xml

```diff
@@ -30,23 +28,34 @@
 			<li>Queue manager.</li>
 		</ul>
 	</description>
+	<categories>
```

Comment:
Should be put on the desktop file only. They are added automatically here with appstream compose.